spec.md

```diff
@@ -151,6 +151,21 @@ Service Broker MAY reject the request with `412 Precondition Failed` and
 provide a message that informs the operator of the API version that is to be
 used instead.
 
+## Extending Resources
+
+Senders of messages defined by this specification MAY include additional
+fields within the JSON objects as extensions. For backwards compatibility
+reasons, new fields MAY be added at any location within the JSON objects,
+however, they SHOULD be placed within the `extensions` property of the JSON
```

Comment:
I don't think the `extensions` property is a good addition. If this becomes an emergent pattern I think you push it into the spec at a later time but this change really limits the integration with existing broker-like solutions. The OSBAPI applied as a mix-in has way more power than forcing the implementor to make OSB the most important part of the API.

Reply:
@n3wscott are you suggesting we go with one of the previous proposals and allow extensions anywhere w/o a formal wrapper?

@pmorie any comments on @n3wscott's comment?

Reply:
I would prefer them to go anywhere without formal wrapper, yes. I would be fine with

> they MAY be placed within the `extensions` property of the JSON

Reply:
I take it back. I am now in support of something like a `extensions` property after talking/thinking about the future of the API.